### DISCUSSION TITLE
8080是哪个组件用到的端口

### DISCUSSION THREAD

**Discussion by zuoHx580** [apache/rocketmq#6815] [Q&A] [answered]
单机部署，我启动broker的时候指定了特定proxy.json进行启动，为啥一直提示fail bind 8080
而且我看日志也确实读到了我新用的端口8180
![image](https://github.com/apache/rocketmq/assets/45234194/b26f6f83-3835-4d95-a10a-6d5475b4d974)

这是我启动的完整命令：sh bin/mqbroker -n localhost:9876 -pc conf/proxy.json --enable-proxy &

**Comment by bugCats** [comment, marked as answer]
remotingListenPort端口，在proxy.json文件中配置。

**Reply by zuoHx580** [reply]
非常感谢
